vulture/utils.py

```diff
@@ -37,6 +38,17 @@ def read_file(filename):
         raise VultureInputException(err)
 
 
+def return_nodes_after_return(tree):
+    try:
+        for i, node in enumerate(tree.body, 0):
+            if isinstance(node, ast.Return):
+                return tree.body[i+1:]
+    except AttributeError:
+        return []
+    else:
+        return return_nodes_after_return(node)
+
```

Comment:
I tried some basic implementations using `walk` and `iter_child_nodes`. Like:
```python
def return_node_after_return(tree):
    """
    Iterate over list of every node present in each level of the ast.
    Return the item just after return statement.
    """
    for parent_node in ast.walk(tree):
        child_nodes = [child_node for child_node in ast.iter_child_nodes(parent_node)]
        for i, node in enumerate(child_nodes):
            if isinstance(node, ast.Return):
                try:
                    return child_nodes[i+1]
                except IndexError:
                    return None
```
But, lies a pitfall: The _if-else_, _try-except_ clauses are a single node. Thus it would be more difficult to list items in the 'same logical level'. This is also sub-optimal performance wise, but I guess, we can improve it.

Is the current approach not feasible?